epictrack-api/src/api/models/staff_elevated_role.py

```diff
@@ -63,6 +64,7 @@ def find_by_elevated_role_id(cls, elevated_role_id: int, include_inactive=False)
     @classmethod
     def find_by_staff_id(cls, staff_id: int, include_inactive=False):
         """Return by staff id."""
+        current_app.logger.info(f"find staff elevated roles by staff_ID: {staff_id}")
```

Comment:
:thinking:

Reply:
Fixed in [a672afa](https://github.com/bcgov/EPIC.track/pull/2552/commits/a672afa2b9909f75f6483273913a511cd99f590d)